app/src/main/java/fr/neamar/kiss/dataprovider/simpleprovider/SearchProvider.java

```diff
@@ -27,7 +27,7 @@
 import fr.neamar.kiss.utils.PackageManagerUtils;
 
 public class SearchProvider extends SimpleProvider {
-    private static final String URL_REGEX = "^(?:[a-z]+://)?(?:[a-z0-9-]|[^\\x00-\\x7F])+(?:[.](?:[a-z0-9-]|[^\\x00-\\x7F])+)+.*$";
+    private static final String URL_REGEX = "^(?:[a-z]+://)?(?:[a-z0-9-]|[^\\x00-\\x7F])+(?:[.](?:[a-z0-9-]|[^\\x00-\\x7F])+)?.*$";
```

Comment:
I had modified it because that was its only use or URL matcher and the URI validation method was in a search instance context and not in a static context. But yes by extracting it to a utility it's probably more suitable

Reply:
Modified @marunjar !